test/cpp/c10d/TCPStoreTest.cpp

```diff
@@ -204,6 +205,50 @@ TEST(TCPStoreTest, testCleanShutdown) {
   clientThread.join();
 }
 
+TEST(TCPStoreTest, testLibUVPartialRead) {
+  int numWorkers = 2;
```

Comment:
Wait, we say two clients here but we only have one client in the test?

Reply:
Oops, that's a typo. Since we did not wait for workers, this was not captured in test.... 🙃

Reply:
I guess what I am trying to say is that you basically only use one client not two. If so why do you even need one extra thread (aka, extra client) if you only use one?

Reply:
Oh I see. That is right, we didn't utilize the client created in the main thread. Good point! I think you're right we can remove the extra thread.